app/src/ui/publish-repository/publish.tsx

```diff
@@ -141,7 +141,7 @@ export class Publish extends React.Component<IPublishProps, IPublishState> {
           selectedIndex={this.state.currentTab}
         >
           <span>GitHub.com</span>
-          <span>Enterprise</span>
+          <span>Enterprise Server</span>
```

Comment:
should this be "GitHub Enterprise Server too"?

Reply:
Agreed, this should be “GitHub Enterprise Server”